Solutions/Recorded Future/Data/Solution_RecordedFuture.json

```diff
@@ -25,7 +25,7 @@
     "Workbooks/Recorded Future - Actively Communicating C&C IPs to DNS Events - Correlation&Threat Hunting.json"
   ],
   "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\Recorded Future",
-  "Version": "2.1.0",
+  "Version": "2.2.1",
```

Comment:
@RecordedFutureOskbo, can you please check if version can be incremented to 2.1.1 rather than 2.2.1, thanks.

Reply:
Yes the version on content hur released a week ago was 2.2.0, this is a copy/backup of the solution file that we copy to the Tools\Create-Azure-Sentinel-Solution\V2\input folder.
![image](https://user-images.githubusercontent.com/104766953/210250700-82d97f65-ddff-4ba5-b033-9ea0c405e084.png)

Reply:
@RecordedFutureOskbo, thanks for the confirmation.